api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java

```diff
@@ -123,38 +123,73 @@ public BatchRecorder newMeasureBatchRecorder() {
     return new NoopBatchRecorder();
   }
 
+  @Override
+  public LabelSet createLabelSet(String k1, String v1) {
+    Utils.checkNotNull(k1, "k1");
+    Utils.checkNotNull(v1, "k1");
+    return NoopLabelSet.INSTANCE;
+  }
+
+  @Override
+  public LabelSet createLabelSet(String k1, String v1, String k2, String v2) {
+    Utils.checkNotNull(k1, "k1");
+    Utils.checkNotNull(v1, "k1");
+    Utils.checkNotNull(k2, "k2");
+    Utils.checkNotNull(v2, "v2");
+    return NoopLabelSet.INSTANCE;
+  }
+
+  @Override
+  public LabelSet createLabelSet(String k1, String v1, String k2, String v2, String k3, String v3) {
+    Utils.checkNotNull(k1, "k1");
+    Utils.checkNotNull(v1, "k1");
+    Utils.checkNotNull(k2, "k2");
+    Utils.checkNotNull(v2, "v2");
+    Utils.checkNotNull(k3, "k3");
+    Utils.checkNotNull(v3, "v3");
+    return NoopLabelSet.INSTANCE;
+  }
+
+  @Override
+  public LabelSet createLabelSet(
+      String k1, String v1, String k2, String v2, String k3, String v3, String k4, String v4) {
+    Utils.checkNotNull(k1, "k1");
+    Utils.checkNotNull(v1, "k1");
```

Comment:
Did you copy-paste all that code?! 😱 
```suggestion
    Utils.checkNotNull(v1, "v1");
```